src/marqo/tensor_search/tensor_search.py

```diff
@@ -769,7 +771,7 @@ def _add_documents_structured(config: Config, add_docs_params: AddDocsParams, ma
                         # 5. load correct media type into memory -> PIL (images), videos (), audio (torchaudio)
                         # 6. if chunking -> then add the extra chunker
 
-                        if isinstance(field_content, str) and not _is_image(field_content):
+                        if not is_image_pointer_field:
```

Comment:
Previous check also required a string?

Reply:
I think the str check is not needed here anymore as we removed the Image.Image